com.github.dictation_toolbox.Caster.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: io.github.dictation_toolbox.Caster
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: python3 -m dragonfly load _caster.py --engine kaldi --no-recobs-messages
```

Comment:
I already mentionned this on matrix.

Reply:
This should be a single command that is the main executable of the application. If it doesn't have one, you can create it.